### PR TITLE
CI: run R CHECK

### PR DESCRIPTION
**Pull request by swo**

### CODE REVIEW

File: uptakeprojection/R/stan_slope.R
Comment (iholmdahl) [inline uptakeprojection/R/stan_slope.R:1]: This file is no longer running for me, causing project_uptake.R (which depends on fit_slopes.R) to fail.

**Review by iholmdahl — CHANGES_REQUESTED** [review]
Overall I think this is fine (thanks for the help with package compliance!). I can't tell which change exactly it is but something in the update to stan_slope.R is causing it to fail when I run the script.

File: uptakeprojection/R/stan_slope.R
Comment (swo) [reply uptakeprojection/R/stan_slope.R:1]: What kind of error message do you get?

Comment (swo) [reply uptakeprojection/R/stan_slope.R:1]: Is this the same as: https://github.com/CDCgov/covid-vaccine-uptake-2023/pull/1#issuecomment-2492587358 ?

Comment (iholmdahl) [reply uptakeprojection/R/stan_slope.R:1]: the message is:

`Error in `mutate()`:
ℹ In argument: `stan = purrr::map(fit_data, stan_slope_summary)`.
Caused by error in `purrr::map()`:
ℹ In index: 1.
Caused by error in `h()`:
! error in evaluating the argument 'object' in selecting a method for function 'extract': 'data' must be a list, environment, or character vector`

when i'm running `project_uptake.R` -- it comes in when `fit_slopes.R` gets to running the `stan_slope.R` function

Comment (swo) [reply uptakeprojection/R/stan_slope.R:1]: This was a problem of `c()` vs. `list()`

**Review by iholmdahl — CHANGES_REQUESTED** [review]
This is working now -- will be fine once it's passing checks

**Review by iholmdahl — APPROVED** [review]

**Pull request merged**